### PR TITLE
hwdb: Fixed Wifi toggling for Haier 7G-Series/JWU (#25293)

### LINKED ISSUE #25293: Haier 7G-Series/JWU: Pressing Wifi toggling function key results in unknown key pressed message
### systemd version the issue has been seen with

252

### Used distribution

Debian sid/unstable

### Linux kernel version used

6.0.0-2-amd64 (6.0.6)

### CPU architectures issue was seen on

x86_64

### Component

udev rule files

### Expected behaviour you didn't see

On the Intel Haswell system HaierComputer 7G-Series/JWU, BIOS KQ122 05/08/2014, pressing the function key to toggle the Wifi device on and off, the desktop environment should give feedback, that the toggling happened.

### Unexpected behaviour you saw

GNOME does not provide any feedback, and Linux logs:

    [ 2177.373205] atkbd serio0: Unknown key pressed (translated set 2, code 0x91 on isa0060/serio0).
    [ 2177.373216] atkbd serio0: Use 'setkeycodes e011 <keycode>' to make it known.
    [ 2177.507512] atkbd serio0: Unknown key released (translated set 2, code 0x91 on isa0060/serio0).
    [ 2177.507528] atkbd serio0: Use 'setkeycodes e011 <keycode>' to make it known.

### Steps to reproduce the problem

_No response_

### Additional program output to the terminal or log subsystem illustrating the issue

_No response_

### PR DESCRIPTION
**Pull request by taniishkaaa**
Fixes #25293

### CODE REVIEW

**Review by yuwata — APPROVED** [review]

File: hwdb.d/60-keyboard.hwdb [hunk 1]
```diff
@@ -523,6 +523,10 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnGeminiDevices:pnNC14V1006:*
 # Genius
 ###########################################################
 
+# HaierComputer 7G-Series
```
Comment (yuwata) [inline hwdb.d/60-keyboard.hwdb]: Hm, I am not familiar with the company, but please add `Haier` section and put it there:
```
###########################################################
# Haier
###########################################################

# 7G-Series
evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHaierComputer:pn7G-Series:*
 KEYBOARD_KEY_91=wlan
```

**Review by yuwata — CHANGES_REQUESTED** [review]

File: hwdb.d/60-keyboard.hwdb [hunk 2]
```diff
@@ -523,6 +523,14 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnGeminiDevices:pnNC14V1006:*
...
 ###########################################################
 
+###########################################################
+# Haier
+###########################################################
+
+# 7G-Series
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHaierComputer:pn7G-Series:*
+ KEYBOARD_KEY_91=wlan
+
```
Comment (yuwata) [inline hwdb.d/60-keyboard.hwdb]: Then, please move the new "Haier" section above "Huawei" section, to make the sections sorted alphabetically.

**Review by yuwata — CHANGES_REQUESTED** [review]

File: hwdb.d/60-keyboard.hwdb (hunk 2 above)
Comment (taniishkaaa) [reply hwdb.d/60-keyboard.hwdb]: > Then, please move the new "Haier" section above "Huawei" section, to make the sections sorted alphabetically.

I think it should be placed above Hewlett-Packard too, to be alphabetically sorted.

Comment (yuwata) [reply hwdb.d/60-keyboard.hwdb]: Yeah. You are right.

File: hwdb.d/60-keyboard.hwdb [hunk 3]
```diff
@@ -523,6 +523,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnGeminiDevices:pnNC14V1006:*
 # Genius
 ###########################################################
 
+
+
```
Comment (bluca) [inline hwdb.d/60-keyboard.hwdb]: spurious newlines

**Review by yuwata — APPROVED** [review]
LGTM, thank you!

**Pull request merged**